x-pack/legacy/plugins/ml/server/models/bucket_span_estimator/polled_data_checker.js

```diff
@@ -12,7 +12,7 @@
 
 import _ from 'lodash';
 
-export function polledDataCheckerFactory(callWithRequest) {
+export function polledDataCheckerFactory(callAsInternalUser) {
```

Comment:
Is it normal that `callWithRequest` was renamed to `callAsInternalUser`? shouldn't it be `callAsCurrentUser` instead? Or did the behavior change? (same question on some other files)

Reply:
It's replacing legacy es plugin `const { callWithInternalUser } = elasticsearchPlugin.getCluster('admin');`  from (call_with_internal_user_factory https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/ml/server/client/call_with_internal_user_factory.js) -  `context.core.elasticsearch.adminClient.callAsInternalUser` is being passed here.